README.md

```diff
@@ -171,25 +173,23 @@ want.
   ]
 }
 ```
+<br />
 
-</details>
-</details>
+#### Text dataset format
 
-<details>
-<summary>Text</summary>
-<br />
-<details>
-<summary>.csv, .xls/.xlsx</summary>
+- dataset must have an **ID column** (`review_id` in the example below) - a column containing identifiers that uniquely identify each row.
+- dataset must have a **text column** (`review` in the example below) that serves as the input set for the prediction task.
+- dataset must have a **label column** (`sentiment` in the example below) that serves as the output for the prediction task.
```

Comment:
```suggestion
- dataset must have a **label column** (`sentiment` in the example below) which you either want to train models to predict or simply find erroneous values in.
```

Reply:
Don't need to adopt my exact suggestions, but I do think the language should be standardized when we are referring to the same concepts (eg. label is the same for all data modalities)